homeassistant/components/demo/backup.py

```diff
@@ -0,0 +1,49 @@
+"""Backup platform for the demo integration."""
```

Comment:
Should we move this to the kitchen sink integration?

Reply:
Yes.